cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -823,6 +728,77 @@ function LogViewerController ($scope, $window, LogViewerStore, myLogsApi, LOGVIE
       stopPoll();
     }
   });
+
+  if (vm.runId) {
```

Comment:
is this the start of the request? Perhaps add some comment to help indicate it? 😁